doc/adrs/XXX_support_representation_details_view.adoc

```diff
@@ -0,0 +1,55 @@
+= ADR-XXX - Add Support for Representation Details View
+
+== Context
+
+Currently, when a user selects a representation, nothing is displayed in the details section.
+Representations already have some metadata (id, name, targetObjectId, kind).
+But users want more information to distinguish their different representations.
+
+== Decision
+
+To address this need, we will enrich the metadata of representations by adding additional information such as a description and an icon.
+These details will be displayed in the details section when a representation is selected, and should be editable by end user.
+
+=== View DSL
+
+* Enhance the `representationDescription` type to add a description to the representation and the ability to define icons
+
+class RepresentationDescription {
+    EString description
+    InterpretedExpression iconExpression
+}
```

Comment:
While I agree on the iconExpression, now that I'm thinking about it, what is this description field supposed to contain? An initial description written by the specifier? Do we need the specifier to provide an initial value? If so why not an expression to make it specific to the representation created. If there's no requirement for an initial value, it does not belong in the view DSL / representation description at all, it's just a field in RepresentationMetadata like the label.